themes/digital.gov/layouts/shortcodes/img-flexible.html

```diff
@@ -0,0 +1 @@
+{{- partial "core/img-flexible.html" (dict "Site" .Site "src" (.Get "src") "small" (.Get "small")) -}}
```

Comment:
Seeing as `small` is a boolean, we could use one as the default and the other as the override.

**For example**
_If small is false, show large. Otherwise always show small._

Let's create a card if it's a lot of re-work.